File: game-admin/src/main/java/com/dkm/problem/entities/bo/ProblemEntity.java
package com.dkm.problem.entities.bo;

import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import kotlin.Metadata;
import kotlin.jvm.internal.Intrinsics;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

@NoArgsConstructor
@AllArgsConstructor
@Data
@TableName("tb_problem")
public final class ProblemEntity {
    @TableId
    private long id;
    @NotNull
    private String problemSubject;
    @Nullable
    private String problemAnswerA;
    @Nullable
    private String problemAnswerB;
    @Nullable
    private String problemAnswerC;
    @Nullable
    private String problemAnswerD;
    @Nullable
    private String problemAnswer;


}

File: game-admin/src/main/java/com/dkm/problem/mapper/bo/ProblemEntity.kt
package com.dkm.problem.mapper.bo

import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName

@TableName("tb_problem")
data class ProblemEntity(
        @TableId
        val id: Long,
        val problemSubject: String,
        val problemAnswerA: String?,
        val problemAnswerB: String?,
        val problemAnswerC: String?,
        val problemAnswerD: String?,
        val problemAnswer: String?


)
